public/images/hubble-captures-vivid-auroras-in-jupiters-atmosphere_28000029525_o~orig.md

```
@@ -0,0 +1,35 @@
+
+Jupiter, the largest planet in the solar system, is best known for its colorful storms, the 
+most famous being the Great Red Spot. Now astronomers have focused on another beautiful feature 
+of the planet, using Hubble's ultraviolet capabilities.
+
+The extraordinary vivid glows shown in the new observations are known as auroras. They are 
+created when high-energy particles enter a planet’s atmosphere near its magnetic poles and 
+collide with atoms of gas. As well as producing beautiful images, this program aims to 
+determine how various components of Jupiter’s auroras respond to different conditions in the 
+solar wind, a stream of charged particles ejected from the sun.
+
+This observation program is perfectly timed as NASA’s Juno spacecraft is currently in the 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)